WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -663,6 +670,11 @@ private extension ProductFormViewController {
         }
     }
 
+    func createPublishBarButtonItem() -> UIBarButtonItem {
+        let updateTitle = NSLocalizedString("Publish", comment: "Action for creating a new Product remotely")
+        return UIBarButtonItem(title: updateTitle, style: .done, target: self, action: #selector(updateProduct))
```

Comment:
Do we really want to call the `updateProduct` method here? Probably the behavior it's the same, but inside `updateProduct` we call also the analytics event `eventLogger.logUpdateButtonTapped()`. This would affect our stats.

Reply:
great catch on the analytics event! I created a different function `publishProduct` for this case, and renamed the functions for saving a product remotely (either from adding/editing) here: https://github.com/woocommerce/woocommerce-ios/pull/2846/commits/7d03e6f5f713f2bce57f4d1127f4f50307cf0c96

Reply:
This is great! 💯